### DISCUSSION TITLE
More modern DOS hardware

### DISCUSSION THREAD

**Discussion by jschwartzenberg** [dosemu2/dosemu2#2274] [Off-topic]
After the thread with some notebooks running DOS (https://github.com/dosemu2/dosemu2/discussions/1800), there's now also a mini-ITX board specifically made for DOS.

https://invidious.privacyredirect.com/watch?v=ogHqmjn6sY4

**Comment by stsp** [comment]
Well, they are proud with their built-in
SB, but there is already https://github.com/crazii/SBEMU
They have AGP slot for video, but where
to get such a card?
IMHO such board needs an ISA slots for
the real antique HW, and it needs a voodoo
card built-in, rather than providing an AGP
slot. Also there should be some OS to DL
and run the DOS games from internet.
Its a cool thing, but I really don't think its
very useful. OS can be written (or installed),
but no ISA slots, no built-in video capable of
3dfx-compatible accel makes me wonder.

**Comment by stsp** [comment]
https://www.youtube.com/watch?v=ogHqmjn6sY4
In case your link doesn't work.
